TASVideos/TagHelpers/CollapsableTagHelper.cs

```diff
@@ -23,14 +23,6 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
 				<a class='collapsed' data-bs-toggle='collapse' {Attr("href", "#" + BodyId)} aria-expanded='false' aria-controls='collapse1' role='button'>
 					{content}
 				</a>
-				<a
-					data-bs-toggle='collapse'
-					class='collapsed btn btn-default btn-xs text-end'
-					{Attr("href", "#" + BodyId)}
-					{Attr("aria-label", $"Expand/Collapse {BodyId}")}
-					aria-expanded='false'
-					role='button'
-				>
```

Comment:
The tag helper had a dark circle icon that's not visible in dark mode, and it was obsoleted by Masterjun icons so I removed the dark icon from the tag helper. And then removed the collapse option from userfile comments lol.